runtime/onert/core/src/compiler/train/pass/Pass.h

```diff
@@ -26,6 +26,7 @@ namespace ir
 namespace train
 {
 class TrainableGraph;
+class TrainingInfo;
```

Comment:
Please remove the forward declaration `class TrainingInfo` below in `onert::compiler::train`?

Reply:
Thank you. Updated :)